### PR TITLE
New Achievements & first aid starter pack

### PR DESCRIPTION
**Pull request by Balferian**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/7406
* **Server Mode**: Re
* **Description of Pull Request**: 
- [x] Uncommented achievements reward.
- [x] Changed start items for Renewal (added first aid pack).
- [x] Added missed First aid / Adventurer boxes (1-95) and few usable items from this boxes.

### CODE REVIEW

File: db/re/item_db_usable.yml [hunk 1]
```diff
@@ -3802,6 +3802,36 @@ Body:
...
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      itemheal rand(27,37),0;
+  - Id: 11615
+    AegisName: Sweet_Potato_
+    Name: Sweet Potato
+    Type: Healing
+    Weight: 30
```
Comment (Atemo) [inline db/re/item_db_usable.yml:3824]:
```suggestion
    Weight: 20
```

File: db/re/item_db_usable.yml [hunk 2]
```diff
@@ -47553,6 +47598,468 @@ Body:
     Script: |
       specialeffect2 EF_POTION_BERSERK;
       sc_start SC_INFINITY_DRINK,1800000,0;
+  - Id: 23484
+    AegisName: Firstaid_Box_5
+    Name: First aid Box (5)
+    Type: Usable
```
Comment (Atemo) [inline db/re/item_db_usable.yml:47604]: EquipLevelMin is missing for all box
```suggestion
    Type: Usable
    EquipLevelMin: 5
```

File: db/re/item_db_usable.yml [hunk 3]
```diff
@@ -47553,6 +47598,468 @@ Body:
...
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 23506,8;
+      getitem 11572,45;
+      getitem 22543,2;
+  - Id: 23575
```
Comment (Atemo) [inline db/re/item_db_usable.yml:47943]: Id order is messed up from here

File: db/re/item_db_usable.yml [hunk 4]
```diff
@@ -47553,6 +47595,493 @@ Body:
...
+      NoCart: true
+      NoStorage: true
+      NoGuildStorage: true
+      NoMail: true
+      NoAuction: true
+    Script: |
+      getitem 11572,45;
+      getitem 22543,2;
+      getitem 23506,8;
+  - Id: 23575
```
Comment (Atemo) [inline db/re/item_db_usable.yml:47959]: For Id order I mean you put the item ID in wrong order

23502
23575
...
23581
23505 (already in db)

Comment (Balferian) [reply db/re/item_db_usable.yml:47959]: oh, thanks, i`ll update it

**Review by Lemongrass3110 — APPROVED** [review]

**Review by aleos89 — APPROVED** [review]

**Review by Atemo — APPROVED** [review]

**Pull request merged**